x/iro/types/iro.pb.go

```diff
@@ -0,0 +1,1663 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: dymensionxyz/dymension/iro/iro.proto
+
+package types
+
+import (
+	fmt "fmt"
+	_ "github.com/cosmos/cosmos-proto"
+	github_com_cosmos_cosmos_sdk_types "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/cosmos/cosmos-sdk/types"
+	_ "github.com/cosmos/gogoproto/gogoproto"
+	proto "github.com/cosmos/gogoproto/proto"
+	github_com_cosmos_gogoproto_types "github.com/cosmos/gogoproto/types"
+	_ "google.golang.org/protobuf/types/known/durationpb"
+	_ "google.golang.org/protobuf/types/known/timestamppb"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+	time "time"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+var _ = time.Kitchen
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+// Params is a module parameters.
+type Params struct {
+	TakerFee    github_com_cosmos_cosmos_sdk_types.Dec `protobuf:"bytes,1,opt,name=taker_fee,json=takerFee,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Dec" json:"taker_fee"`
+	CreationFee github_com_cosmos_cosmos_sdk_types.Int `protobuf:"bytes,2,opt,name=creation_fee,json=creationFee,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Int" json:"creation_fee"`
+	// Minimum plan duration
+	// when creating a plan, the plan duration should be greater than or equal to this value
+	// plan duration is from the start time to the pre-launch time
+	MinPlanDuration time.Duration `protobuf:"bytes,3,opt,name=min_plan_duration,json=minPlanDuration,proto3,stdduration" json:"min_plan_duration"`
+	// Incentive plan minimum parameters
+	// when creating a plan, the incentive plan parameters should be greater than or equal to this value
+	IncentivePlanMinimumParams IncentivePlanParams `protobuf:"bytes,4,opt,name=incentive_plan_minimum_params,json=incentivePlanMinimumParams,proto3" json:"incentive_plan_minimum_params"`
```

Comment:
this name seems very committed to the fact that every param which is incentive related will be a minimum.
I think it's cleaner to simply incroporate the min/max in the nested var names. 

so `MinStartTimeAfterSettlement` and `MinNumEpochsPaidOver` so when we see it also in genesis and queries it's easy to understand this param without the wrapping context of min.